xamarin/Xamarin.Forms
# [iOS] Fixed wrong groups CollectionView header size

### Description of Change ###

The groups header was taking the size corresponding to:

`availableSpace / Spans`

### Issues Resolved ### 

- fixes #8557

### API Changes ###
 
 None

### Platforms Affected ### 

- iOS

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

#### Before
![69101110-804c7a80-0a3d-11ea-824f-322c7dd8c19d](https://user-images.githubusercontent.com/6755973/69142080-59865680-0ac6-11ea-8aa5-ebcaa0bfffd2.png)

#### After
<img width="412" alt="Captura de pantalla 2019-12-20 a las 11 43 14" src="https://user-images.githubusercontent.com/6755973/71249710-1c5bf100-231e-11ea-9b53-3ad95091e59e.png">

### Testing Procedure ###
Launch Core Gallery and navigate to the Issue8557. If the group header size occupies the entire width (the text should appear centered), the test has passed.

### PR Checklist ###
